acquia_cms.profile

```diff
@@ -330,3 +330,21 @@ function acquia_cms_update_8001() {
   $config->set('enable_export_filtering', TRUE);
   $config->save(TRUE);
 }
+
+/**
+ * Prepares variables for maintenance page templates.
+ *
+ * Default template: maintenance-page.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - content - An array of page content.
+ *
+ * @see template_preprocess_maintenance_page()
+ */
+function cohesion_theme_preprocess_maintenance_page(array &$variables) {
```

Comment:
@chandan-singh7929 Why this hook is cohesion_theme. This should be `acquia_cms`. Right ?

Reply:
Done